README.md

```diff
@@ -1 +1,28 @@
 # jbCrypt
+
+## Lab Class 14
+
+## Feature Tasks 
+
+- Create a database.
+  Create a table that models users with a username, id, and hashed password.
+  Create a table that models posts with a postId, userId and text content.
+  Create a table with a 1-to-many relationship between users and posts.
+  Create login/signup page.
+  Authenticate the site so only logged-in users can see posts.
+  Unauthenticated users accessing protected areas of site should be redirected to login/signup page.
+  Use Bcrypt to hash all passwords stored on the site.
+
+### Limitations 
+- the app is completed but the blog has not been completed and implemented. Intent is to be able to generate blogs for the blogpost topic. Ultimately there is more to be done 
+
+#### Self Notes 
+
+- Keep track of Routes and understand where redirects can happen
+- Two controllers can co-exist within the app seems like good practice for each controller to keep track of the models it is responsible for
+- application.properties need to point to the actual running instance of the database application 
+- need to tinker or ask why the new database was not able to connect, thinking that two databases where taking up the same port will test more or ask 
```

Comment:
All databases on the same server can use that server's port. If the new database is on a new server, that might be the problem. If it's on the same server, double-check:
- The database has actually been created in postgres/PGAdmin (I forget this a lot)
- The database name is spelled exactly right in your `application.properties`
- 'spring.jpa.hibernate.dll-auto=` is set to `create`, not `update`